### PR TITLE
Added Avatar in Student Section (Google Code-in task)

### PR DESCRIPTION
**Pull request by ezralazuardy**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://gci18.ezralazuardy.tk/#students)

## Description / Changes

Added Student: Ezra Lazuardy

### Before:
before
### After:
after

- - - - - - - - - - - -

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-gci18.fossasia.org
```
[the remark this answers is not in the document]
Comment (ezralazuardy) [reply CNAME]: Travis build passed

[the remark this answers is not in the document]
Comment (nikit19) [reply CNAME]: Please revert this change. Rest looks good to me !

[the remark this answers is not in the document]
Comment (ezralazuardy) [reply CNAME]: ok, it's done! :)

**Comment by ezralazuardy** [comment]
@nikit19 , changes have been reverted

**Pull request merged**